pennylane/templates/state_preparations/basis_qutrit.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Contains the BasisStatePreparation template.
+"""
+
+import pennylane as qml
+from pennylane.operation import Operation, AnyWires
+
+
+class QutritBasisStatePreparation(Operation):
+    r"""
```

Comment:
Please add the `QutritBasisStatePreparation` template to `docs/introduction/templates.rst` so its included in the documentation

Reply:
How is the png image included in `:figure:` generated?

Reply:
That's not a super set thing. But you can just run this to get an image representing the template, and then save it as a png and add it to the folder:

```
import pennylane as qml

dev = qml.device("default.qutrit", wires=3)

@qml.qnode(dev)
def circuit(basis_state):
    qml.QutritBasisStatePreparation(basis_state, wires=range(3))
    return qml.state()

basis_state = [1, 0, 2]
qml.draw_mpl(circuit)(basis_state)
```

You're also welcome to create a more illustrative circuit using `TShift` operators instead of `QutritBasisStatePreparation` and draw that instead if you prefer, akin to the circuit shown for the `BasisState` template :)

Reply:
I added this graph:

![image](https://github.com/PennyLaneAI/pennylane/assets/57567043/5ed24b6f-d80c-438c-9eca-d4f05bd2c1e4)

but I think one of these two look "friendlier" 😆 

![image](https://github.com/PennyLaneAI/pennylane/assets/57567043/f71753c7-b746-4282-9958-f12bf986afdb)

![image](https://github.com/PennyLaneAI/pennylane/assets/57567043/06b56110-f452-4a72-9902-122d3e214427)

But I guess that would imply changing the other ones to keep similar format, no?

Reply:
So something is off because the documentation check is failing but the documentation in readthedocs looks just fine.

Reply:
I agree, the new plotting format is friendlier 😅 But keeping the format consistent makes sense. 

Sphinx can be a little finicky for the docs build, this is the error it seems to be reporting (you just have to scroll up endlessly until you eventually find its complaint, unfortunately it doesn't give any details in the final `build failed` message):

![image](https://github.com/PennyLaneAI/pennylane/assets/38584660/48920ba0-98ab-4da8-b9d1-beb9cff3418b)

Reply:
Omg, thank you! I searched "error" and could not find the source of the problem. And even the warning on your screenshot showed "line 27" which is not even where the error was 😕  Thank you for pointing it out!! 

To build the docs locally for testing, I understand from this https://docs.pennylane.ai/en/stable/development/guide/documentation.html that I should have the interfaces requirements installed, right?

Reply:
I see, once again: thank you 😄 
I believe this was the last fix from this batch of comments then!